### DISCUSSION TITLE
Are DDoS allowed now?

### DISCUSSION THREAD

**Discussion by tarkhil** [community/community#13371] [Other Feature Feedback, Questions, & Ideas]
Hello

I did not find it in Policy, but apparently DDoS tools are allowed now by GitHub.

Can't anyone please point me at the related changes in Policy? Are any DDoS allowed or anti-Russian only?

**Comment by suprstarrd** [comment]
I don't think DDoS tools are allowed, period! Althouogh, I'm not GitHub staff, so the best place to confirm/report this definitely isn't here; go contact [GitHub support](https://github.com/support)

**Reply by tarkhil** [reply]
I've reported three repos, and cannot even find my tickets by numbers. So I think that AT LEAST IN SOME CASES GitHub now allow malware, DDoS tools and so on.

**Comment by MioVisman** [comment]
Similarly, all requests for repositories with DDOS tools used against Russian sites remain unanswered.
There are only unsubscribes from the robot with gratitude.

GitHub did you declare war on Russia?

**Reply by suprstarrd** [reply]
To be clear: I support Ukraine, but will try my best to make these responses as neutral as possible (I don't want this discussion to get too political or off-topic; I fear that could make this heated!)

Also to be clear: I'm not GitHub staff. So, don't take anything I say as 100% truth.

> Similarly, all requests for repositories with DDOS tools used against Russian tools remain unanswered.

If you haven't submitted any requests for any other DDoS tool that isn't against a Russian tool, you have nothing to compare. I'm not saying you haven't, because I don't know. However, until I can see something to compare it with, I strongly doubt that this is a case of "Russian sites are fine to DDoS", and more so a case of "we just haven't reviewed any DDoS repository, period".

> GitHub did you declare war on Russia?

That would be strange, considering their responses to the conflict [here](https://github.blog/2022-03-02-our-response-to-the-war-in-ukraine/) and [here](https://github.com/github/feedback/discussions/12042#discussioncomment-2276190), which in comparison to other companies is very relaxed, being neither pro-Russia nor pro-Ukraine.

**Reply by suprstarrd** [reply]
Made some edits to make my position more clear.

**Reply by tarkhil** [reply]
> Made some edits to make my position more clear.

Well... It's a good idea to make some repo with DDoS tools.
